tasks/icons/Edit.tsx

```diff
@@ -0,0 +1,7 @@
+import { Edit } from "lucide-react";
+
+const EditIcon = () => {
+  return <Edit color="black" size={24} />;
```

Comment:
It looks quite good, but I think there is no need for an extra component for the icon.